### PR TITLE
HID Braille Driver: add more mappings for arrow keys

### PR DESCRIPTION
**Pull request by michaelDCurran**
### Link to issue number:
None

### Summary of the issue:
Reported by Orbit, The Standard HID Braille driver does not provide mappings to the arrow keys  as they would expect.
Currently the joystick can be used for arrow keys and enter, but not the dpad. Some Braille devices, such as the Orbit Reader, only have a dpad and no joystick.
Also, currently space+dot1 and space+dot4 are mapped to nextLine and previousLine braille commands. Manufacturers would prefer that these gestures map to up and down arrow, so that they also work in lists and on the Desktop etc.

### Description of changes 
When using a Braille Display via the Standard HID braille driver, the dpad can be used to emulate the arrow keys and enter. Also space+dot1 and space+dot4 now map to up and down arrow respectively.

### Testing strategy:
Connected to an Orbit Reader 40 with the Standard HID braille driver:
* While on the Desktop, ensured that the direction buttons of the dpad move the focus in the direction expected, and that dpad enter activates the focused icon.
Ensured that space+dot1 and space+dot4 emulate up and down arrow and move the focus up and down the icons.

### Known issues with pull request:
None known.

### Change log entries:
New features
Changes
- When using a Braille Display via the Standard HID braille driver, the dpad can be used to emulate the arrow keys and enter. Also space+dot1 and space+dot4 now map to up and down arrow respectively.
Bug fixes
For Developers

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

File: source/brailleDisplayDrivers/hidBrailleStandard.py [hunk 1]
```diff
@@ -220,22 +220,39 @@ def display(self, cells: List[int]):
...
 				"br(hidBrailleStandard):space+dot1+dot3+dot4+dot5",
 			),
 			"kb:shift+tab": ("br(hidBrailleStandard):space+dot1+dot3",),
 			"kb:tab": ("br(hidBrailleStandard):space+dot4+dot6",),
 			"kb:alt": ("br(hidBrailleStandard):space+dot1+dot3+dot4",),
 			"kb:escape": ("br(hidBrailleStandard):space+dot1+dot5",),
-			"kb:enter": ("br(hidBrailleStandard):joystickCenter"),
+			"kb:enter": (
+				"br(hidBrailleStandard):joystickCenter",
+				"br(hidBrailleStandard):dpadCenter",
```
Comment (seanbudd) [inline source/brailleDisplayDrivers/hidBrailleStandard.py:254]: should dot8 be in this list?

File: source/brailleDisplayDrivers/hidBrailleStandard.py [hunk 2]
```diff
@@ -220,22 +220,39 @@ def display(self, cells: List[int]):
 				"br(hidBrailleStandard):panRight",
 				"br(hidBrailleStandard):rockerDown",
 			),
-			"braille_previousLine": ("br(hidBrailleStandard):space+dot1",),
```
Comment (seanbudd) [inline source/brailleDisplayDrivers/hidBrailleStandard.py:223]: should we still have commands from previous and next line?

File: source/brailleDisplayDrivers/hidBrailleStandard.py (hunk 1 above)
Comment (michaelDCurran) [reply source/brailleDisplayDrivers/hidBrailleStandard.py:254]: "bk:dot8" (enter) and "bk:dot7" (backspace) are mapped to scripts in globalCommands.py already as they apply to all braille displays, so don't need to be explicitly mentioned in any braille driver.

File: source/brailleDisplayDrivers/hidBrailleStandard.py (hunk 2 above)
Comment (michaelDCurran) [reply source/brailleDisplayDrivers/hidBrailleStandard.py:223]: braille_nextLine and kb_downArrow generally do the same thing, at least by default. If however you tether braille to the review cursor, then braille_nextLine will move the review cursor up and down by line, but I think this is a fairly rare usecase, and certainly none of the braille manufacturers I'm talking with seem to care. In future we could consider some kind of toggle command on the braille display to switch between panning and moving by line or something. But it seems the expectation is that joystick / dpad acts like pressing the arrow keys on the keyboard.

**Review by seanbudd — APPROVED** [review]

**Pull request merged**